modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,181 @@
+
+.Procedure
+
+. Create the namespace for the first {istio} control plane called `istio-system-1`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-1
+----
+
+. Add the following label to the first namespace, which will be used with the Istio `discoverySelector`:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-1 istio-discovery=mesh-1
+----
+
+. Create a YAML file named `istio-1.yaml` with the name `mesh-1` and the discoverySelector `mesh-1`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-1
+spec:
+  namespace: istio-system-1
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: mesh-1
+----
+
+. Create the first `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-1.yaml
+----
+
+. Create a namespace for the second Istio control plane called `istio-system-2`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-2
+----
+
+. Add the following label to the second namespace, which will be used with the Istio `discoverySelector`:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-2 istio-discovery=mesh-2
+----
+
+. Create a YAML file named `istio-2.yaml`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-2
+spec:
+  namespace: istio-system-2
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: mesh-2
+----
+
+. . Create `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-2.yaml
+----
+
+. To restrict workloads in `mesh-1` from communicating freely with unencrypted traffic between meshes, deploy a `PeerAuthentication` resource to enforce mutual TLS (mTLS) traffic within the `mesh-1` data plane. Apply the `PeerAuthentication` resource in the `istio-system-1` namespace by using a configuration file, such as `peer-auth-1.yaml`:
++
+[source,terminal]
+----
+$ oc apply -f peer-auth-1.yaml
+----
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: security.istio.io/v1
+kind: PeerAuthentication
+metadata:
+  name: "mesh-1-peerauth"
+  namespace: "istio-system-1"
+spec:
+  mtls:
+    mode: STRICT
+----
+
+. Deploy a policy for workloads in the `istio-system-2` namespace to only accept mutual TLS traffic `peer-auth-2.yaml`:
++
+[source,terminal]
+----
+$ oc apply -f peer-auth-2.yaml
+----
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: security.istio.io/v1
+kind: PeerAuthentication
+metadata:
+  name: "mesh-2-peerauth"
+  namespace: "istio-system-2"
+spec:
+  mtls:
+    mode: STRICT
+----
+
+.Verfication
+
+. Verify that both of the {istio} control planes deploy and run. This can be done by validating that the `Istiod` pod is successfully running in each {istio} system namespace.
```

Comment:
Pod name should be in lower-case.
```suggestion
. Verify that both of the {istio} control planes deploy and run. This can be done by validating that the `istiod` pod is successfully running in each {istio} system namespace.
```